EIPS/eip-3540.md

```diff
@@ -164,6 +164,7 @@ For clarity, the *container* refers to the complete account code, while *code* r
 4. `CODECOPY`/`CODESIZE`/`EXTCODECOPY`/`EXTCODESIZE`/`EXTCODEHASH` keeps operating on the entire *container*.
 5. The input to `CREATE`/`CREATE2` is still the entire *container*.
 6. The size limit for deployed code as specified in [EIP-170](./eip-170.md) and for initcode as specified in [EIP-3860](./eip-3860.md) is applied to the entire *container* size, not to the *code* size. This also means if initcode validation fails, it is still charged the EIP-3860 `initcode_cost`.
+7. When an EOF1 contract performs a `DELEGATECALL` the target contract has to be EOF1. If it is not EOF1 (e.g. it is EOF0 or EOF2), the `DELEGATECALL` exceptionally halts. Hence, (among other things) all the gas passed to the `DELEGATECALL` is consumed and 0 is pushed onto the stack. As it is not valid EOF1, `DELEGATECALL` to an empty container also fails.
```

Comment:
Maybe makes sense to mention that target address still becomes warm (is added to `accessed_addresses`, if not already there) even if it doesn't contain EOF1 code

Reply:
Thanks @gumb0 , excellent suggestion. Will add that.